pkg/models/search.go

```diff
@@ -1,6 +1,42 @@
-package search
+package models
```

Comment:
Would it work to place this in a `searchmodel` package under `pkg/services/search`? I guess this was moved because of a circular dependency situation, but maybe we should still try to keep things within their service, even if that service aren't nicely structured?

Reply:
Moving DTOs between packages is expensive (i.e. modifies too many files and makes PR bigger), I'd vote for type aliases as a temporary measure, and yes, I'd prefer to have DTOs closer to their corresponding domain packages, rather than a big "models" package for everything.

Reply:
just aliased for now for starmodel, we can move the file later.